Spec/RealtimeClientPresence.swift

```diff
@@ -9,11 +9,83 @@
 import Quick
 import Nimble
 
+func addRandomMemberToChannel(channelName: String, options: ARTClientOptions, done: ()->()) -> [ARTRealtime] {
+    let client = ARTRealtime(options: options)
+    let channel = client.channels.get(channelName)
+    channel.attach() { _ in
+        channel.presence.enterClient(AblyTests.newRandomString(), data: nil)
+        done()
+    }
+    return [client]
+}
+
 class RealtimeClientPresence: QuickSpec {
     override func spec() {
         describe("Presence") {
 
+            // RTP1
+            context("ProtocolMessage bit flag") {
+
+                it("when no members are present") {
+                    let options = AblyTests.commonAppSetup()
+                    options.autoConnect = false
+                    let client = ARTRealtime(options: options)
+                    client.setTransportClass(TestProxyTransport)
+                    client.connect()
+                    defer { client.close() }
+                    let channel = client.channels.get("test")
+                    channel.attach()
+
+                    expect(channel.state).toEventually(equal(ARTRealtimeChannelState.Attached), timeout: testTimeout)
+
+                    let transport = client.transport as! TestProxyTransport
+                    let attached = transport.protocolMessagesReceived.filter({ $0.action == .Attached })[0]
+
+                    expect(attached.flags & 0x1).to(equal(0))
+                    expect(attached.isSyncEnabled()).to(beFalse())
+                    expect(channel.presence.syncComplete).to(beFalse())
+                    expect(channel.presenceMap.isSyncComplete()).to(beFalse())
+                }
+
+                it("when members are present") {
```

Comment:
I would agree, please add 250 present members, and expect 3 SYNC pages of members.